### ISSUE TITLE
please create an new api for  reboot appium server by http request

### ISSUE THREAD

**Issue by xian-crazy** [appium/appium#13266]
## The problem

please create an new api for  reboot appium server by http request

some times, the appium server should be reboot at start ot end of test

i just excute command line like "kill -9 xxxx;appium xxxx"  with runtime  class in java, it is very poor;
i hope appium server can give a more “Elegant” method like a http api to reboot appium server

thanks very very very much!

**Comment by mykola-mokhnach** [comment]
I don't think this would be very practical. Such implementation is complicated, since it must be operating system-specific, as we would need to use the system command line interpreter directly. Also it would be necessary to copy the process environment and command line arguments of the current process, which is also not very trivial task. And the third issue - if something bad has happened and the server is frozen then it won't be able to handle a request for restart anyway.
